<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_Form</class>
 <widget class="QWidget" name="settings_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>480</width>
     <height>260</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="mainSettings">
    <attribute name="title">
     <string>Main</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Default </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="windowSettings">
    <attribute name="title">
     <string>Window</string>
    </attribute>
    <widget class="QCheckBox" name="trayCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>400</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimize application to tray</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Display">
    <attribute name="title">
     <string>Display</string>
    </attribute>
    <widget class="QLabel" name="languageLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>User interface language:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="languagesBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>19</y>
       <width>200</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>English (Англійська)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ukranian (Українська)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Russian (Російська)</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
